src/database/migrations/20240716111210-update-user-id-to-string.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		try {
+			// Change columns to STRING
+			await queryInterface.changeColumn('availabilities', 'user_id', {
+				type: Sequelize.STRING,
```

Comment:
@nevil-mathew check is there any test case we have ?  if its present, do we need to change in mock data in that case
@nevil-mathew API doc request and response check if there any change required